pkg/commands/environment.go

```diff
@@ -2,17 +2,19 @@ package commands
 
 import (
 	"errors"
+	"path/filepath"
 
 	environment "github.com/ministryofjustice/cloud-platform-cli/pkg/environment"
+	log "github.com/sirupsen/logrus"
+	"k8s.io/client-go/util/homedir"
 
 	"github.com/MakeNowJust/heredoc"
 	"github.com/spf13/cobra"
 )
 
-var (
-	module        string
-	moduleVersion string
-)
+var module, moduleVersion string
+
+var optFlags environment.Options
```

Comment:
As the `environment` package is large, can you please add a comment as to the purpose of these globals.